ext/bcmath/tests/number/operators/short_syntax.phpt

```diff
@@ -0,0 +1,103 @@
+--TEST--
+BcMath\Number operator short syntax
+--EXTENSIONS--
+bcmath
+--FILE--
+<?php
+
+echo "========== add ==========\n";
+$num = new BcMath\Number(100);
+$num_old = $num;
```

Comment:
Why do you have these num_old assignments?

Reply:
Fixed in https://github.com/php/php-src/pull/13741/commits/fc320a9a0f19c9811f7b446e69a057ea240e77ba
I wanted to make sure it was an immutable object, but I forgot to print it in the test results